target/linux/mvebu/config-4.9

```diff
@@ -279,6 +285,8 @@ CONFIG_MAGIC_SYSRQ=y
 CONFIG_MANGLE_BOOTARGS=y
 CONFIG_MARVELL_PHY=y
 CONFIG_MDIO_BOARDINFO=y
+CONFIG_MDIO_BUS_MUX=y
+CONFIG_MDIO_BUS_MUX_GPIO=y
```

Comment:
These are left over from the patch set I used as my base, I probably should have tidied up earlier.

I assumed that MDIO bus mux had been added for good reason and was somehow necessary to control both the 88E6176 switch and 88E1514 PHY on the same MDIO bus, but both seem to work fine on an image compiled without.

I am away from my Omnia now, but I will do some testing later to test all of these options and remove as many as possible. I suspect some will return in a future patch to add support for the Omnia's I2C-attached microcontroller for RGB LED control, but of course it is better to logically group options with the code that requires them rather than letting cruft build for 'just in case' selections.